x-pack/plugins/ml/public/application/components/help_popover/help_popover.scss

```
@@ -0,0 +1,13 @@
+.mlHelpPopover__panel {
+  max-width: $euiSize * 30;
+}
+
+.mlHelpPopover__content {
+  @include euiYScrollWithShadows;
+  max-height: 40vh;
+  padding: $euiSizeM;
```

Padding with the popover header doesn't appear to match the padding of the content. Assuming you want 8px padding for both, I suggest the following.

```suggestion
  padding: $euiSizeS;
```